### DISCUSSION TITLE
Failing to Clone Repository

### DISCUSSION THREAD

**Discussion by shockley327** [community/community#109381] [New to GitHub]
Hey All -
I am trying to clone an AWS repository and while I have all of the AWS CLI tools installed. It is failing to clone the repository. I have PowerUser access so permissions shouldn't be an issue. Anyone have any idea why I can't clone any repositories?

**Comment by MEClouds** [comment]


**Reply by queenofcorgis** [reply]
@MEClouds
We’ve clarified our stance on using generative AI tools like ChatGPT within our Community via [this announcement](https://github.com/orgs/community/discussions/69006). Please review the guidelines to ensure your post meets them as failure to adhere to those rules can result in action taken by our moderator team.  You can read our updated [Code of Conduct](https://github.com/community/community/blob/baa5cd9020ef3f2bbf86dca1191d9c90f9b0822d/CODE_OF_CONDUCT.md#reasonable-use-of-ai-generated-content) and the [announcement](https://github.com/orgs/community/discussions/69006) for more details. Thank you for helping us maintain an authentic and beneficial space for everyone.
